tests/pyproject.toml

```diff
@@ -8,3 +8,53 @@ exclude = "/(\\.direnv|\\.eggs|\\.git|\\.hg|\\.mypy_cache|\\.nox|\\.tox|\\.venv|
 # https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html
 multi_line_output = 3
 profile = "black"
+
+[tool.pylint.main]
+
+# Use multiple processes to speed up Pylint. Specifying 0 will auto-detect the
+# number of processors available to use, and will cap the count on Windows to
+# avoid hangs.
+jobs = 0
+
+score = false
+
+# Pickle collected data for later comparisons.
+persistent = false
+
+# Minimum Python version to use for version dependent checks. Will default to the
+# version used to run pylint.
+py-version = "3.12"
```

Comment:
do we have to remember to update it when upgrading python?

Reply:
If it defaults to the running version maybe we should remove this then. I removed most of the template already heh.